### ISSUE TITLE
[bug] [accessibility] screen reader does not read most content of cosmic settings

### ISSUE THREAD

**Issue by phoekjelle** [pop-os/cosmic-settings#1763]
When opening the cosmic store, the screenreader only reads the title (i think it's called)

<img width="948" height="1028" alt="Image" src="https://github.com/user-attachments/assets/4eaba444-3181-4ec8-af07-9b98aed90506" />
_only reading «Network en wifi» in this window_

I dont think that is the purpus of the screen reader, there probebly is some documentation on what the screen should read, and in what order, so that people without the ability to read, can listen to the screen reader and use their flipping computer
